### DISCUSSION TITLE
Gene annotation file (GTF) matching index file used for mapping?

### DISCUSSION THREAD

**Discussion by Nico282828** [gagneurlab/drop#531] [Q&A]
Dear DROP community,
  I would like to ask if the gene annotation file (e.g. gencode v29) and fasta file we provide in the config file have to match those that were used during STAR index file generation for mapping.

  Thank you!

Best Wishes,
Nico

**Comment by vyepez88** [comment]
Hi, the files do not have to exactly match the ones you used for mapping. Be sure, however, that the fasta file matches the genome build and chr style and the gtf file matches the genome build.

**Comment by Nico282828** [comment]
Thank you so much !
